### ISSUE TITLE
Carrier Strike Group

### ISSUE THREAD

**Issue by tillmas** [R0flC0pter/FF-Pretense-Causasus#2]
Add a Tarawa, A Forrestal and one of the Super Carriers. 

Add escorts and recovery tankers. 

Ensure that they route (preferably on separate routes. 

Once this is complete add a separate issue to modify zones to create a follow zone that recognizes them as places where we can sign up for missions. 

Documented by Fisc.

**Comment by tillmas** [comment]
Added in 1.3.6.4.  Also added trigger zones to follow each carrier, but have not linked them into mission sign up.

Added new issue.
